pytorch_lightning/accelerators/accelerator.py

```diff
@@ -243,16 +243,3 @@ def __setstate__(self, d):
 
     def on_save(self, checkpoint):
         return checkpoint
-
-
-# TODO: allow user to compare with string even internaly we shall use these Enum to prevent typos...
-class BackendType(Enum):
-    DP = 'dp'
-    DDP = 'ddp'
-    DDP2 = 'ddp2'
-    DDP_SPAWN = 'ddp_spawn'
-    # decuple distrib and device
-    DDP_CPU = 'ddp_cpu'
-    HOROVOD = 'horovod'
-    # this is rather device
-    TPU = 'tpu'
```

Comment:
can we avoid enums please?
it's always impossible to figure out how to import them etc...

Reply:
Id they inherit `str`, then they are optional for users. And we can still use the enum internally for nicer code (my opinion)

Reply:
as they inherit from str, you can compare with string so resolving the issue you faced last time.. see the examples